modular_citadel/code/modules/client/loadout/backpack.dm

```diff
@@ -23,6 +23,16 @@
 	category = SLOT_IN_BACKPACK
 	path = /obj/item/toy/plush/lampplushie
 
+/datum/gear/toy/plush/borgplushie/secdrake
+	name = "SecDrake Plushie"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/toy/plush/borgplushie/secdrake
+
+/datum/gear/toy/plush/borgplushie/meddrake
+	name = "MediDrake Plushie"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/toy/plush/borgplushie/meddrake
+
```

Comment:
personally its better to scrap loadout plushies and make it a choice beacon so people can start with /any/ loadout plushie (apart from those with special effects)

Reply:
I am a terrible coder, I have never messed with choice beacon code, while that would reduce bloat on loadout list this would, however, prevent people from having the choice of having multiple plushies if that is what they wanted. the plushies are added so they can be obtained.